packages/app/src/templates/components/template-form/byte-length.validator.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { RegisterOptions } from "react-hook-form/dist/types"
+import { intl } from "Renderer/utils/intl"
+
+export const byteLengthValidator = (): RegisterOptions => ({
+  required: intl.formatMessage({ id: "module.templates.required" }),
+  validate: (value: string): string | undefined => {
+    if (new Blob([value]).size > 469) {
```

Comment:
Maybe `469` could be moved to some const (maybe e.g. `messages` module)? Seems to me that this might be a shared const and it belong more to the "messages" module (as this is the maximum size of the SMS in bytes).